src/WarcraftLegacies.Source/Quests/Goblin/QuestRatchet.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using MacroTools.ControlPointSystem;
+using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.FactionBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.ObjectiveSystem.Objectives.UnitBased;
+using MacroTools.QuestSystem;
+using static War3Api.Common;
+
+
+namespace WarcraftLegacies.Source.Quests.Goblin
+{
+  /// <summary>
+  /// The Goblins can acquire Ratchet.
+  /// </summary>
+  public sealed class QuestRatchet : QuestData
+  {
+    private readonly List<unit> _rescueUnits;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestRatchet"/>.
+    /// </summary>
+    /// <param name="noggenfogger">Must be brought somewhere to complete the quest.</param>
+    public QuestRatchet(LegendaryHero noggenfogger) : base("Ratchet Port",
+      "The Port of Ratchet would be a great expansion of our trade empire.",
```

Comment:
Port isn't a proper noun in this context so no need to capitalize.

Reply:
Lowercase p